Kickstarter-iOS/Views/Controllers/BetaToolsViewController.swift

```diff
@@ -135,16 +185,21 @@ internal final class BetaToolsViewController: UIViewController {
     self.present(alert, animated: true, completion: nil)
   }
 
-  private func showEnvironmentActionSheet() {
+  private func showEnvironmentActionSheet(sourceViewIndex: Int) {
+    guard let sourceView = self.tableView
```

Comment:
It looks like this is still a bit off on iPad...I wonder if we could present this from the centre of the cell (since the cell is the element that you're interacting with (not the label) and therefore we can tap anywhere on it).

I tried quickly fixing this but wasn't really working just right...wonder if it's worth spending a bit time on this...it should be as simple as adjusting the coordinates of the view or maybe even better passing `sourceRect` to our convenience function to present popover.

This shows how to popover overlays the label and is technically aligning to `0, 0` coordinate of the label (ideally tis would be `midX` & `midY` of the cell maybe with some optional offset as well)
<img width="1029" alt="Screen Shot 2019-06-17 at 10 10 25 AM" src="https://user-images.githubusercontent.com/387596/59624082-b7171100-90ea-11e9-8218-14b355196ffd.png">

Reply:
How about we just change the type of alert to `.alert` instead of `.actionSheet` for iPad, this looks better to me and we also wouldn't need to care about the `sourceView`:

<img width="50%" alt="Screen Shot 2019-06-19 at 10 33 46 AM" src="https://user-images.githubusercontent.com/3735375/59787183-c4183980-927d-11e9-8692-ac8ddffcb33b.png">

Reply:
I already updated it to pass the `sourceRect`, but I'm cool with either solution.

Reply:
My preference is `.alert` but it wouldn't say it's a strong feeling.

Reply:
Updated!